### DISCUSSION TITLE
Repository

### DISCUSSION THREAD

**Discussion by Jitendra7653** [community/community#112087] [New to GitHub]
I accidentally deleted the repo yesterday. When I try to recover it says contact support , please only
Repo link :- Jitendra7653/Thumbnail-changer-bot2

**Comment by queenofcorgis** [comment]
Hi there!

If the repository was part of a fork network - you will need to contact Support to restore it. [Check out our documentation for more details](https://docs.github.com/en/repositories/creating-and-managing-repositories/restoring-a-deleted-repository#about-repository-restoration).
